cluster/validation.go

```diff
@@ -731,3 +738,19 @@ func getClusterVersion(version string) (semver.Version, error) {
 	}
 	return parsedVersion, nil
 }
+
+// warnWeaveDeprecation prints a deprecation warning if version is higher than 1.27
+func warnWeaveDeprecation(k8sVersion string) error {
+	version, err := util.StrToSemVer(k8sVersion)
+	if err != nil {
+		return fmt.Errorf("failed to translate: %s to semver notation: %w", k8sVersion, err)
```

Comment:
Probably want to make this more uniform, see https://github.com/rancher/rke/blob/c2395d3451087f864ec46af8226604ceb7caaf07/hosts/tunnel.go#L70